sdk/typescript/src/faucet/faucet.ts

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
this is just what was in index.ts before, making all index.ts files just a list of explicit re-exports so its easier to see what is being publicly exported